release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2865,3 +2865,25 @@ link:https://access.redhat.com/solutions/6487101[{product-title} 4.8.19 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-8-20"]
+=== RHBA-2021:4574 - {product-title} 4.8.20 bug fix update
+
+Issued: 2021-10-15
```

Comment:
Is this the correct date?

Reply:
Nope. It should be 2021-10-16. I will change it. Thanks for catching it.

Reply:
Done! It's updated now.

Reply:
This looks good! Let me know when you're ready for it to be merged.